### PR TITLE
[FIX] - Empty links "Develop" section

### PR DESCRIPTION
**Pull request by nhussein11**
This PR aims to fix empty links in the "Develop" section

### CODE REVIEW

File: .snippets/code/tutorials/blockchains/custom-blockchains/build-a-blockchain/add-trusted-nodes/chainspec-head.html [hunk 1]
```diff
@@ -1,18 +1,4 @@
 <div id="termynal" data-termynal>
   <span data-ty="input"><span class="file-path"></span>head customSpec.json</span>
-  <span data-ty>
```
Comment (0xLucca) [inline .snippets/code/tutorials/blockchains/custom-blockchains/build-a-blockchain/add-trusted-nodes/chainspec-head.html:3]: This formatting must be rolled back

File: .snippets/code/tutorials/blockchains/custom-blockchains/build-a-blockchain/add-trusted-nodes/chainspec-tail.html [hunk 2]
```diff
@@ -1,82 +1,5 @@
 <div id="termynal" data-termynal>
   <span data-ty="input"><span class="file-path">tail -n 78 customSpec.json</span></span>
-  <span data-ty>{
```
Comment (0xLucca) [inline .snippets/code/tutorials/blockchains/custom-blockchains/build-a-blockchain/add-trusted-nodes/chainspec-tail.html:3]: This formatting must be rolled back

File: .snippets/code/tutorials/blockchains/custom-blockchains/build-a-parachain/connect-a-parachain/connect-a-parachain-3.html [hunk 3]
```diff
@@ -1,8 +1,5 @@
 <div id="termynal" data-termynal>
-  <span data-ty="input"><span class="file-path"></span>./target/release/parachain-template-node build-spec \
```
Comment (0xLucca) [inline .snippets/code/tutorials/blockchains/custom-blockchains/build-a-parachain/connect-a-parachain/connect-a-parachain-3.html:2]: This formatting must be rolled back

File: .snippets/code/tutorials/blockchains/custom-blockchains/build-a-parachain/connect-a-parachain/connect-a-parachain-5.html [hunk 4]
```diff
@@ -1,17 +1,5 @@
 <div id="termynal" data-termynal>
-  <span data-ty="input"><span class="file-path"></span>./target/release/parachain-template-node \
```
Comment (0xLucca) [inline .snippets/code/tutorials/blockchains/custom-blockchains/build-a-parachain/connect-a-parachain/connect-a-parachain-5.html:2]: This formatting must be rolled back

**Review by 0xLucca — CHANGES_REQUESTED** [review]
The auto formatter broke multiple code snippets, fix that before mergingg

File: develop/blockchains/deployment/generate-chain-specs.md [hunk 5]
```diff
@@ -131,6 +131,6 @@ The raw chain specification can be used to initialize the genesis storage for a
 
 After generating a chain specification, you can use it to initialize the genesis storage for a node. Refer to the following guides to learn how to proceed with the deployment of your blockchain:
 
-- [**Obtain Coretime**](TODO:update-path) - learn how to obtain the necessary coretime configuration to synchronize your blockchain’s timestamping and enhance its performance
-- [**Deployment**](TODO:update-path) - explore the steps required to deploy your chain specification, ensuring a smooth launch of your network and proper node operation
-- [**Maintenance**](TODO:update-path) - discover best practices for maintaining your blockchain post-deployment, including how to manage upgrades and monitor network health
\ No newline at end of file
+- [**Obtain Coretime**](/tutorials/blockchains/custom-blockchains/build-a-parachain/acquire-a-testnet-slot/#obtain-coretime){target=\_blank} - learn how to obtain the necessary coretime configuration to synchronize your blockchain’s timestamping and enhance its performance
```
Comment (themacexpert) [inline develop/blockchains/deployment/generate-chain-specs.md:134]: I fixed a few links here, just double check that the obtain coretime is the one you want! @nhussein11

**Review by themacexpert — APPROVED** [review]

File: develop/blockchains/deployment/generate-chain-specs.md (hunk 5 above)
Comment (eshaben) [reply develop/blockchains/deployment/generate-chain-specs.md:135]: This works for now. Added a comment with a todo to change it once we have an obtain coretime guide

**Review by 0xLucca — APPROVED** [review]
LGTM!

**Pull request merged**